packages/shared/common/src/api/platform/Platform.ts

```diff
@@ -1,9 +1,15 @@
 import { Crypto } from './Crypto';
+import { Encoding } from './Encoding';
 import { Filesystem } from './Filesystem';
 import { Info } from './Info';
 import { Requests } from './Requests';
 
 export interface Platform {
+  /**
+   * The interface for performing base64 operations.
+   */
+  encoding: Encoding;
```

Comment:
```suggestion
  encoding?: Encoding;
```

So we don't have to implement it for server side right now.